rllib/agents/maml/maml.py

```diff
@@ -199,9 +197,17 @@ def validate_config(self, config: TrainerConfigDict) -> None:
     @override(Trainer)
     def get_default_policy_class(self, config: TrainerConfigDict) -> Type[Policy]:
         if config["framework"] == "torch":
+            from ray.rllib.agents.maml.maml_torch_policy import MAMLTorchPolicy
+
             return MAMLTorchPolicy
+        elif config["framework"] == "tf":
+            from ray.rllib.agents.maml.maml_tf_policy import MAMLDynamicTFPolicy
+
+            return MAMLDynamicTFPolicy
```

Comment:
Dont have to do this in this PR, just a thought: Should we just call these `TFPolicy` instead of `DynamicTFPolicy`. I feel like all TF-ones are dynamic (loss) now.